roles/sft-server/molecule/default/converge.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Converge
+  hosts: all
+  tasks:
+    - name: include molecule variables
+      include_vars: shared_vars.yml
+
+    - debug: var=sft_fqdn
+
+    - name: check INSTANCE_UUID is set
+      assert:
+        that:
+          - molecule_instance_id != ""
+        msg: >
+          [ERR] Please run:
+            export INSTANCE_UUID=$RANDOM
+          then re-run this playbook.
+
+    # Normally we assume DNS to be created externally-to-ansible.
+    # Here, we create a DNS record using ansible only for molecule tests.
+    - name: A records
+      route53:
+        command:   create
+        zone:      "{{ sft_root_domain }}"
+        record:    "{{ sft_fqdn }}"
+        type:      A
+        ttl:       60
+        value:     "{{ hostvars[groups['all'][0]].ansible_default_ipv4.address }}"
```

Comment:
I'm not sure how molecule works internally or if it even matters here, but in general, you should be able to just `ansible_default_ipv4.address`. Tho, I'm not sure when the variables is being read, since the task is `delegate_to: localhost`. (So, when in doubt, ignore me)

Reply:
Yeah, we want IP address of the host created by molecule. `ansible_default_ipv4.address` will get IP address of localhost, which would be bad.  I think this is the only way to get it.